src/main.c

```diff
@@ -2605,6 +2601,10 @@ static int finish_open_listener(int fd, int apply_bpf_filter, int cpus, const ch
     if (fd < 0)
         goto Error;
 
+    if (conf.globalconf.bp.nic_count > 0) {
+        h2o_busypoll_bind_interface(fd, iface);
```

Comment:
this change on its own is likely not going to be sufficient -- you'll need to rework the logic in `open_listener` to move more of it to after this call in `finish _open_listener`

Reply:
I've reverted this change in a53af73e0dc92341ce8af3d170b198851913350b, i don't think i can easily move the `bind()` out of `open_listener`, which means i can't move the setting of `so_reuseport` and since `so_bindtodevice` has to be set before that, it has to remain in `open_listener`